### PR TITLE
Fix zeroes not being displayed in formatted strings

### PR DESCRIPTION
**Pull request by rlivsey**

### CODE REVIEW

**Comment by redbugz** [comment]
:+1:  we have some places we display zero as well, so this will affect us as well.

**Comment by pgherveou** [comment]
I ran into this issue on our app too, can a new version be published on npm?

**Comment by jasonmit** [comment]
Ping @ericf @caridy

File: tests/index.js [hunk 1]
```diff
@@ -110,6 +110,15 @@ describe('IntlMessageFormat', function () {
 
             expect(output).to.equal('My name is Anthony Pipkin.');
         });
+
+        it('should not ignore zero values', function() {
+            var mf = new IntlMessageFormat('I am {age} years old.');
```
Comment (ericf) [inline tests/index.js]: Why not use `{age, number}` in something like this?

File: src/compiler.js [hunk 2]
```diff
@@ -156,7 +156,7 @@ function StringFormat(id) {
 }
 
 StringFormat.prototype.format = function (value) {
-    if (!value) {
+    if (value === undefined || value === null) {
```
Comment (ericf) [inline src/compiler.js]: What about `false`?

**Comment by rlivsey** [comment]
Closing as `{foo, number}` does the job, and `{foo, plural … }` renders zeroes fine when using `#`.

**Pull request closed without merging**